etc/DependencyInstaller.sh

```diff
@@ -478,7 +486,17 @@ while [ "$#" -gt 0 ]; do
         -dev|-development)
             echo "The use of this flag is deprecated and will be removed soon"
             ;;
+        -base)
+            option="base"
+            ;;
+        -common)
+            option="common"
+            ;;
         -local)
+            if [[ $(id -u) == 0 ]]; then
+                echo "Error: cannot install locally if you are root or using sudo" >&2
+                exit 1
+            fi
             export PREFIX="${HOME}/.local"
             ;;
         -prefix=*)
```

Comment:
Should `-prefix` also check for root/sudo?

Reply:
I don't think so, the problem with using sudo and local was that the $HOME value changed from ~ to /root but I believe if the user specified a defined prefix from the start that won't be determined in runtime then it shouldn't be a problem. 
For example, running `sudo ./etc/DependencyInstaller.sh -prefix=$HOME/local` the prefix will be set to ~/local.

If you agree, I can allow it to work normally for -prefix.

Reply:
Honestly, I believe if we only have -prefix indicating that it can be set with $HOME/local we can get rid of -local. Then we wouldn't need all of the new changes (new flags).

Reply:
Let's allow in prefix then and keep the local as it was a suggestion/request from Austin https://github.com/The-OpenROAD-Project/OpenROAD/pull/2405#pullrequestreview-1153810442